docs/content/modeling/user-groups.mdx

```diff
@@ -175,31 +174,31 @@ In order to assign `member`s of a `team` a relation to a `document`, we can crea
   ]}
 />
 
-### 04. Checking An Individual Member's Access To An Object
+### 04. Check An Individual Member's Access To An Object
 
-Now that we have:
+Now that you have:
 
-- a relationship tuple indicating that **alice** is an `member` of **team:writers**
-- a relationship tuple indicating that **members of team:writers are editors of document:meeting_notes.doc**
+- a relationship tuple indicating that `alice` is an `member` of `team:writers`
+- a relationship tuple indicating that `members` of `team:writers` are editors of `document:meeting_notes.doc`
 
-This means that if we \*<ProductConcept section="what-is-a-check-request" linkName="check" />**\*is alice an editor of document:meeting_notes.doc**? We would get the following:
+The \*<ProductConcept section="what-is-a-check-request" linkName="check" />\ `is alice an editor of document:meeting_notes.doc` returns the following:
 
 <CheckRequestViewer user={'user:alice'} relation={'editor'} object={'document:meeting_notes.doc'} allowed={true} />
 
-The chain of resolution becomes:
+The chain of resolution is:
 
-- **alice** is `member` of **team:writers**
-- `member`s of **team:writers** are `editor`s of **document:meeting_notes**
-- therefore, **alice** is `editor` of **document:meeting_notes**
+- `alice` is `member` of `team:writers`
+- `member`s of `team:writers` are `editor`s of `document:meeting_notes`  
+- therefore, `alice` is `editor` of `document:meeting_notes`
 
 :::caution
-**Note:** When creating relationship tuples for <ProductName format={ProductNameFormat.ShortForm}/> make sure to use unique ids for each object and user within your application domain. We're using first names and simple ids to just illustrate an easy-to-follow example.
+**Note:** When creating relationship tuples for <ProductName format={ProductNameFormat.ShortForm}/>,  use unique ids for each object and user in your application domain. This example uses first names and simple ids as an easy-to-follow example.
```

Comment:
```suggestion
**Note:** When creating relationship tuples for <ProductName format={ProductNameFormat.ShortForm}/>,  use unique ids for each object and user in your application domain. This example uses first names and simple ids as a suggested example.
```

I think we usually try to shy away from implied ease of use.

Reply:
```suggestion
**Note:** When creating relationship tuples for <ProductName format={ProductNameFormat.ShortForm}/>,  use unique ids for each object and user in your application domain. This example uses names to make it simpler to follow.
```

Not a big fan of the repeated `example`. Also we're not suggesting they use this, in fact we're strongly suggesting they don't :)